### PR TITLE
Integrate cargokit subtree

### PR DESCRIPTION
**Pull request by bookshiyi**
Related Issue: #95 
Related PR: #99 

| Target  | In example | In project |
| ------- | ---------- | ---------- |
| Android | ✅         | ✅         |
| iOS     | ✅         | ✅         |
| macOS   | ✅         | ✅         |
| Linux   | ✅   | ✅   |
| Windows | ✅  | ✅  |

Because `CMake` and `Gradle` script of `cargokit` currently has some problems with the incoming manifest path(Cocoapods no problem) [Comment](https://github.com/cunarist/rust-in-flutter/pull/99#issuecomment-1681630182) , in order to integrate cargokit, choose `subtree` instead of `submodule` as a transition solution.

### CODE REVIEW

**Comment by temeddix** [comment]
Happy to see that this is going forward :D

**Comment by temeddix** [comment]
I made an issue at cargokit: https://github.com/irondash/cargokit/issues/10

**Comment by bookshiyi** [comment]
> I made an issue at cargokit: [irondash/cargokit#10](https://github.com/irondash/cargokit/issues/10)

Submodule method is also going forward😄

**Comment by bookshiyi** [comment]
More tests are welcome.

**Comment by temeddix** [comment]
Whoa, great job. Let me start the review :)

File: example/windows/runner/flutter_window.cpp [hunk 1]
```diff
@@ -31,6 +31,11 @@ bool FlutterWindow::OnCreate() {
     this->Show();
   });
 
+  // Flutter can complete the first frame before the "show window" callback is
+  // registered. The following call ensures a frame is pending to ensure the
+  // window is shown. It is a no-op if the first frame hasn't completed yet.
+  flutter_controller_->ForceRedraw();
```
Comment (temeddix) [inline example/windows/runner/flutter_window.cpp:37]: May I ask you where this line came from? Is it from new Flutter 3.13 default template or something?

Comment (bookshiyi) [reply example/windows/runner/flutter_window.cpp:37]: Yes, when I developing windows target today,  this code was added automatically after command `flutter run -d windows` executed.
I think it is latest version dart/flutter SDK did that.

Comment (temeddix) [reply example/windows/runner/flutter_window.cpp:37]: Oh I see :D Everything else looks good

**Review by temeddix — APPROVED** [review]

**Comment by temeddix** [comment]
Looks good! I only made some slight changes to the code. I will merge it right away :)

**Pull request merged**